service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/cmo/UploadCMOController.java

```diff
@@ -78,29 +70,6 @@ public CallbackResponse handleReviewCMO(@RequestBody CallbackRequest request) {
         return respond(caseDetails);
     }
 
-    @Deprecated // TODO: 19/10/2020 Delete when FPLA-2019 is toggled on
-    @PostMapping("/mid-event")
```

Comment:
can be removed from zap audit file

Reply:
dont see this one removed from definitions, was it removed earlier?

Reply:
Would have been removed during the toggle I think, when the old event definitions was replaced with the new event definitions

Reply:
Will double check to be sure